pages/1.11/release-notes/1.11.6/index.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.6
+title: Release Notes for 1.11.6
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.11.6
+---
+
+DC/OS 1.11.6 was released on September 25, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.6/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+
+DC/OS 1.11.6 includes the following components:
+- Apache Mesos 1.5.x [change log](https://github.com/apache/mesos/blob/19d17cec3e797758e76c081efb68867d440ed4d3/CHANGELOG).
```

Comment:
The Mesos URL should be - https://github.com/apache/mesos/blob/5a7ad47e8fc1a14101e47a29eb8e7e2a20d959c5/CHANGELOG